components/alert/alerts.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+
+
+:root{
+    --primary-color:#456568;
+    --secondary-color:whitesmoke;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+
+li{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
+.nav-container{
+    display: flex;
+    align-items: center;
+    background-color: var(--primary-color);
+    font-family: 'Roboto', sans-serif;
+    font-weight: bold;
+}
+.branding{
+    display: flex;
+    align-items: center;
+    /* gap:1rem; */
+    margin-left: 2rem;
+    padding: 1rem;
+    
+}
+.brand_image{
+    width: 3.5rem;
+    height: 3.5rem;
+}
+
+.brand_name{
+    margin-left: 1rem;
+    font-size: 2rem;
+    color: var(--secondary-color);
+}
+.navbar{
+    margin-left: auto;
+}
+
+.nav-items{
+    display: flex;
+    align-items: center;
+    margin-right: 3rem;
+}
+
+.item-1{
+    margin-right: 3rem;
+}
+
+.doc{
+    color: var(--secondary-color);
+}
+
+.github-icon{
+    color: var(--secondary-color);
+    font-size: 2rem;
+}
+
+/* SIDE-BAR */
+.wrapper{
```

Comment:
I see repeated styles (similar styles in docs.css). You can maybe combine them in a single CSS sheet.